src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -424,4 +437,4 @@ public NetworkPeer Connect(IPEndPoint endpoint)
             return node;
         }
     }
-}
+}
```

Comment:
Did you remove the extra lines at the bottom?

Reply:
I think this was again a review comment on an earlier commit... there are no extra lines at the bottom ;)